daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -456,6 +458,20 @@ abstract class TestCase(testCaseXML: NodeSeq, val parent: DFDLTestSuite)
 
   final def isCrossTest(implString: String) = implString != "daffodil"
 
+  val removeLineColInfo = new RewriteRule {
+    override def transform(n: Node) = n match {
+      case e: Elem => {
+        e.copy(attributes = n.attributes.filter(a =>
+        (a.key != XMLUtils.COLUMN_ATTRIBUTE_NAME) &&
+        (a.key != XMLUtils.LINE_ATTRIBUTE_NAME) &&
+        (a.key != XMLUtils.FILE_ATTRIBUTE_NAME)))
+      }
```

Comment:
Should we also make sure the the namespace is the DAFINT namespace?  "column", "line", and "file" are pretty generic, I could see this easily removing attributes in some other namespace with the same name that shouldn't be removed.

Reply:
Will fix, they have a prefixedKey method that I didn't see the first time around